src/span.h

```diff
@@ -248,9 +248,8 @@ template <typename T>
 T& SpanPopBack(Span<T>& span)
 {
     size_t size = span.size();
-    ASSERT_IF_DEBUG(size > 0);
```

Comment:
Why drop the assert? Though I have no idea who/what tests with `-DDEBUG` anyway.